.github/workflows/AssetsUpdater.yml

```diff
@@ -0,0 +1,37 @@
+name: Assets Updater
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "**.ts"
+      - "**.json"
```

Comment:
Shouldn't `*.ts` do the trick already or am I wrong?

Reply:
Copied from other workflows we have, we should change at a later point